src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -266,6 +208,11 @@ public double encoderPosition() {
         return climbFixedMotor.getSelectedSensorPosition();
     }
 
+    @Log
+    public boolean isSolenoidExtended() {
+        return solenoid.get() == SolenoidState.EXTEND.value;
```

Comment:
Please add a check for if `solenoid` is null (This is causing null pointer exceptions because `Hardware.climbAngle` isn't initialized).

Reply:
I missed `extendArmSolenoid()` and `retractArmSolenoid()`, which should also have checks.